config/config.js

```diff
@@ -29,5 +29,7 @@ module.exports = Object.freeze({
 
   gitCommit: process.env.GIT_COMMIT,
 
-  docsLink: process.env.DOCS_LINK || 'https://dev.ambrosus.com/'
+  docsLink: process.env.DOCS_LINK || 'https://dev.ambrosus.com/',
+
+  requestSizeLimit: process.env.REQUEST_SIZE_LIMIT || '5mb'
```

Comment:
@danihodovic at first proposed 10mb. Maybe stick to this as a default value?

Reply:
I'm fine with either. We can modify it down the road.

Reply:
Actually, can we stick to 1mb?